packages/scrapers-v2/src/types.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Catalog Representations:
+ *
+ * { document { <section> ... } }
+ * { section <desc> <comment> <row[]> }
+ * { row { <textRow | courseRow | multiCourseRow> ...} }
+ *    textRow:
+ *      { areaHeader <comment> <hour> }
+ *      { commentRow <comment> <hour> }
+ *      { subHeader  <comment> <hour> }
+ *
+ *    courseRow:
+ *      { courseRow <code> <hour> }
+ *      { orCourseRow <code> <hour> }
+ *
+ *    multiCourseRow:
+ *      { andCourseRow <hour> { <code, description> ... } }
```

Comment:
update this comment to include RANGEs or delete it (now that we have actual data definitions)